src/actions/WindowActions.js

```diff
@@ -761,6 +766,113 @@ export function fetchTopActions(windowType, docId, tabId) {
   };
 }
 
+const getAPIUrl = function({ windowId, docId, tabId, rowId, path }) {
+  let documentId = docId;
+
+  if (!docId && rowId) {
+    documentId = rowId[0];
+  }
+
+  return `${config.API_URL}/window/${windowId}${
+    documentId ? `/${documentId}` : ''
+  }${rowId && tabId ? `/${tabId}/${rowId}` : ''}/${path}`;
+};
+
+export function callAPI({ windowId, docId, tabId, rowId, target, verb, data }) {
```

Comment:
I'd give it a more meaningful name related to what it actually does, where and what for.

Reply:
It's actually a meaningful name for what it does. I will just add a proper description for the function as I missed that